plugin/src/test/java/io/fabric8/maven/plugin/mojo/build/MappingConfigTest.java

```diff
@@ -0,0 +1,36 @@
+package io.fabric8.maven.plugin.mojo.build;
+
+import java.io.File;
+import org.apache.maven.plugin.testing.MojoRule;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+
+@Ignore("I have not been able to make it run due classpath problems. "
```

Comment:
If you agree, I will open a new issue so we can investigate later when we do the deep refactor to see if we can clarify all dependencies and make test run.

Reply:
I think you could just add it as an it test into below `it/` where you provide a fragment with an new type, add this type to the pom.xml and then assert on the generated resources whether your given fragment has been added. 'guess this is simpler.

Reply:
Maybe we should remove the test then for now as we have an IT now ? Don't think we need to keep it for fixing it later.